docs/plugins/repository-s3.asciidoc

```diff
@@ -153,6 +153,13 @@ settings belong in the `elasticsearch.yml` file.
    https://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/services/s3/AmazonS3Builder.html#setPathStyleAccessEnabled-java.lang.Boolean-[AWS
    documentation] for details). Defaults to `false`.
 
+`disable_chunked_encoding`::
```

Comment:
As written, these docs are in the middle of the docs for the `path_style_access` setting. Could you move them so they're not?